app/jobs/request_verification_code_text_message_job.rb

```diff
@@ -0,0 +1,12 @@
+class RequestVerificationCodeTextMessageJob < ApplicationJob
+  def perform(phone_number:, locale:)
+    TextMessageVerificationCodeService.request_code(
+      phone_number: phone_number,
+      locale: locale
+    )
+  end
+
+  def priority
+    PRIORITY_HIGH
```

Comment:
how come email is higher priority than text message?

Reply:
I copied my priority from PYA. I have changed Text message to match